File: week7/methodoverride.java
class Animal{
    public void eat(){
          System.out.println("i am the parent class");
          System.out.println("i eat to live");
    }
}
class dog extends Animal{
    public void eat(){
       System.out.println("i am a child class");
       System.out.println("i like to eat chicken");  
    }
}
public class methodoverride{
     public static void main(String[] args) {
        dog obj= new dog();
        obj.eat();
    }
}

File: week7/methodoverride.cpp
#include<iostream>
using namespace std;
class Animal{
    public:
     void eat(){
        cout<<"i am a parent"<<endl;
        cout<<"i eat to live"<<endl;
    }
};
class dog: public Animal{
    public:
     void eat(){
       cout<<"i am a child"<<endl;
       cout<<"i like to eat chicken"<<endl;;  
    }
};
int main(){

        dog obj;
        obj.eat();
    
}
